### DISCUSSION TITLE
No more manual configuration

### DISCUSSION THREAD

**Discussion by tafano85** [Koenkk/zigbee2mqtt#8969] [General]
Today I tried to add a new device manually to HomeAssistant, and I noticed that all manual configuration scripts from documentation are... gone!
Then I spotted the commit "Remove manual HA configuration.".
Why, @Koenkk ???? WHY?!?!?!? 😭 

I already wasted hours trying to make automatic configuration work.
Manual HA configuration is my only way to survive! 💪

**Comment by Koenkk** [comment]
I've removed it because it was creating too much maintenance and not fully correct. If many people want to have it back I can investigate that.

**Reply by tafano85** [reply]
Thank you very much for your feedback.
I hope for a come-back of the manual config scripts in the documentation (or equivalent), sooner or later.

**Reply by SaturnusDJ** [reply]
I hope it will be back as well. Reason is that I like to have full control over configurations.

Maybe it is possible to generate it automatically based on device info/specs?
